apps/newsletters-api/src/app/routes/rendering-templates.ts

```diff
@@ -1,14 +1,21 @@
 import type { FastifyInstance } from 'fastify';
-import { makeErrorResponse, makeSuccessResponse } from '../responses';
+import type { NewsletterData } from '@newsletters-nx/newsletters-data-client';
+import { getEmailRenderingHost } from '../../apiDeploymentSettings';
+import { newsletterStore } from '../../services/storage';
+import {
+	makeErrorResponse,
+	makeSuccessResponse,
+	mapStorageFailureReasonToStatusCode,
+} from '../responses';
 
 export type RenderingTemplate = {
 	id: string;
 	status: string;
 	title: string;
 };
 
-const TEMPLATES_LIST_URL =
-	'https://email-rendering.guardianapis.com/info/templates/';
+const NEWSLETTER_RENDER_URL = `${getEmailRenderingHost()}/data-article/render-template`;
+const TEMPLATES_LIST_URL = `${getEmailRenderingHost()}/info/templates/`;
 
```

Comment:
We could avoid evaluating that twice with an assign

```suggestion
const EMAIL_RENDERING_HOST = getEmailRenderingHost();
const NEWSLETTER_RENDER_URL = `${EMAIL_RENDERING_HOST}/data-article/render-template`;
const TEMPLATES_LIST_URL = `${EMAIL_RENDERING_HOST}/info/templates/`;

```